distrobox-enter

```diff
@@ -156,7 +156,7 @@ fi
 # This is not necessary on docker as it is already handled
 # in this way.
 if [ -z "${container_manager#*podman*}" ] &&
-	[ -S "/run/user/$(id -ru)/podman/podman.sock" ] &&
+	[ -S "/run/user/$(id -ru || :)/podman/podman.sock" ] &&
```

Comment:
We can remove the `|| :` if this fails for any reason, it's better to fail generally (if `id` fails there must be some problems anyway :smile:

Reply:
Sadly removing it it causes shellcheck to complain `distrobox-enter:159:20: note: Consider invoking this command separately to avoid masking its return value (or use '|| true' to ignore). [SC2312]` any tip on how to avoid it?

Reply:
Aha! That's new :smile:  it was introduces in 0.8.0 :smile: 

For now we can suppress it, with a simple comment before the if
`# shellcheck disable=SC2312`

I'll decide later if this check will be disabled or not in the future :+1: